command/agent/job_endpoint_test.go

```diff
@@ -171,6 +171,36 @@ func TestHTTP_JobsRegister(t *testing.T) {
 	})
 }
 
+func TestHTTP_JobsRegister_ACL(t *testing.T) {
```

Comment:
I added a comment to make it clear, we are just testing that the token gets parsed and threaded through correctly.